.github/workflows/ci.yml

```
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ "dev" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)